src/DesignCompile/CompileHelper.cpp

```diff
@@ -1815,18 +1823,22 @@ UHDM::tf_call* CompileHelper::compileTfCall(DesignComponent* component, const Fi
     name = fC->SymName(fC->Child(dollar_or_string));
   } else if (leaf_type == slImplicit_class_handle) {
     NodeId handle = fC->Child(dollar_or_string);
-    if (fC->Type(handle) == slSuper_keyword) {
-      name = "super.";
```

Comment:
Where is this happening now?

Reply:
It's handled here: https://github.com/alainmarcel/Surelog/blob/master/src/DesignCompile/CompileExpression.cpp#L2500, then is actually compiled in these cases: https://github.com/alainmarcel/Surelog/blob/master/src/DesignCompile/CompileExpression.cpp#L1246.